app/main.py

```diff
@@ -8,16 +8,15 @@ def __init__(self, name: str, age: int) -> None:
 
 
 def create_person_list(people_list: list) -> list:
-    # Create all Person instances
     person_instances = [
         Person(person["name"], person["age"])
         for person in people_list
     ]
     for person_dict in people_list:
         person_instance = Person.people[person_dict["name"]]
         if "wife" in person_dict and person_dict["wife"]:
-            person_instance.wife = Person.people[person_dict["wife"]]
+            person_instance.wife = Person.people.get(person_dict["wife"])
         if "husband" in person_dict and person_dict["husband"]:
-            person_instance.husband = Person.people[person_dict["husband"]]
+            person_instance.husband = Person.people.get(person_dict["husband"])
```

Comment:
I meant instead of condition `"wife" in person_dict and person_dict["wife"]` you can use get because it does the same thing ;)